Lib/pathlib.py

```diff
@@ -814,9 +814,10 @@ class _PathBase(PurePath):
     __bytes__ = None
     __fspath__ = None  # virtual paths have no local file system representation
 
-    def _unsupported(self, method_name):
-        msg = f"{type(self).__name__}.{method_name}() is unsupported"
-        if isinstance(self, Path):
+    @classmethod
+    def _unsupported(cls, method_name):
+        msg = f"{cls.__name__}.{method_name}() is unsupported"
+        if issubclass(cls, Path):
```

Comment:
Could this be it's own PR?

Reply:
As a standalone PR it's not very well motivated: all the callers of `_unsupported` are instance methods, and so there's no reason to change it to a classmethod. Before I was a core dev I found it pretty impossible to land such PRs, as they're easily regarded as churn.

Reply:
On a first glance it seemed unrelated to the core function of this PR, but I see why you've included it in this PR now, so no objections.